pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -3,999 +3,999 @@ source: pdg/src/main.rs
 
 g {
-	n[0]: &_61 _    => _60 @ bb29[7]:  fn exercise_allocator;
-	n[1]: copy n[0] => _59 @ bb29[8]:  fn exercise_allocator;
-	n[2]: copy n[1] => _58 @ bb29[10]: fn exercise_allocator;
-	n[3]: copy n[2] => _1  @ bb0[0]:   fn printf;            
-}
-nodes_that_need_write = []
-
-g {
-	n[0]:  alloc      _     => _74  @ bb41[2]:  fn exercise_allocator;
-	n[1]:  copy       n[0]  => _1   @ bb42[3]:  fn exercise_allocator;
-	n[2]:  copy       n[1]  => _79  @ bb42[7]:  fn exercise_allocator;
-	n[3]:  offset[0]  n[2]  => _78  @ bb42[7]:  fn exercise_allocator;
-	n[4]:  field.0    n[3]  => _    @ bb43[1]:  fn exercise_allocator;
-	n[5]:  addr.store n[4]  => _    @ bb43[1]:  fn exercise_allocator;
-	n[6]:  copy       n[1]  => _81  @ bb43[6]:  fn exercise_allocator;
-	n[7]:  offset[1]  n[6]  => _80  @ bb43[6]:  fn exercise_allocator;
-	n[8]:  field.0    n[7]  => _    @ bb44[1]:  fn exercise_allocator;
-	n[9]:  addr.store n[8]  => _    @ bb44[1]:  fn exercise_allocator;
-	n[10]: copy       n[1]  => _83  @ bb44[6]:  fn exercise_allocator;
-	n[11]: offset[2]  n[10] => _82  @ bb44[6]:  fn exercise_allocator;
-	n[12]: field.0    n[11] => _    @ bb45[1]:  fn exercise_allocator;
-	n[13]: addr.store n[12] => _    @ bb45[1]:  fn exercise_allocator;
-	n[14]: copy       n[1]  => _85  @ bb45[6]:  fn exercise_allocator;
-	n[15]: offset[3]  n[14] => _84  @ bb45[6]:  fn exercise_allocator;
-	n[16]: field.0    n[15] => _    @ bb46[1]:  fn exercise_allocator;
-	n[17]: addr.store n[16] => _    @ bb46[1]:  fn exercise_allocator;
-	n[18]: copy       n[1]  => _97  @ bb49[15]: fn exercise_allocator;
-	n[19]: offset[0]  n[18] => _96  @ bb49[20]: fn exercise_allocator;
-	n[20]: field.0    n[19] => _95  @ bb51[2]:  fn exercise_allocator;
-	n[21]: addr.load  n[20] => _    @ bb51[2]:  fn exercise_allocator;
-	n[22]: copy       n[1]  => _97  @ bb49[15]: fn exercise_allocator;
-	n[23]: offset[1]  n[22] => _96  @ bb49[20]: fn exercise_allocator;
-	n[24]: field.0    n[23] => _95  @ bb51[2]:  fn exercise_allocator;
-	n[25]: addr.load  n[24] => _    @ bb51[2]:  fn exercise_allocator;
-	n[26]: copy       n[1]  => _97  @ bb49[15]: fn exercise_allocator;
-	n[27]: offset[2]  n[26] => _96  @ bb49[20]: fn exercise_allocator;
-	n[28]: field.0    n[27] => _95  @ bb51[2]:  fn exercise_allocator;
-	n[29]: addr.load  n[28] => _    @ bb51[2]:  fn exercise_allocator;
-	n[30]: copy       n[1]  => _97  @ bb49[15]: fn exercise_allocator;
-	n[31]: offset[3]  n[30] => _96  @ bb49[20]: fn exercise_allocator;
-	n[32]: field.0    n[31] => _95  @ bb51[2]:  fn exercise_allocator;
-	n[33]: addr.load  n[32] => _    @ bb51[2]:  fn exercise_allocator;
-	n[34]: copy       n[1]  => _106 @ bb59[7]:  fn exercise_allocator;
-	n[35]: copy       n[34] => _105 @ bb59[8]:  fn exercise_allocator;
-	n[36]: free       n[35] => _104 @ bb59[9]:  fn exercise_allocator;
+	n[0]: &_61 _    => _60 @ bb29[7]:  fn exercise_allocator; _60 = &raw const (*_61)                              
```

Comment:
Does `&_61` represent `AddrOfLocal(_61)`?  If so, this node is also incorrect - it should be a `Copy` from `_61` instead.

Reply:
that's fixed in #533